stdlib/influxdata/influxdb/secrets/get.go

```diff
@@ -0,0 +1,38 @@
+package secrets
+
+import (
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/interpreter"
+	"github.com/influxdata/flux/semantic"
+	"github.com/influxdata/flux/values"
+)
+
+const GetKind = "get"
+
+func init() {
+	flux.RegisterPackageValue("influxdata/influxdb/secrets", GetKind, GetFunc)
+}
+
+// GetFunc is a function that calls Get.
+var GetFunc = makeGetFunc()
+
+func makeGetFunc() values.Function {
+	sig := semantic.NewFunctionPolyType(semantic.FunctionPolySignature{
+		Parameters: map[string]semantic.PolyType{
+			"key": semantic.String,
+		},
+		Required: semantic.LabelSet{"key"},
+		Return:   semantic.Tvar(1),
```

Comment:
This doesn't need to be a tvar, we can fully specify the type as an object with a `secretKey` property.